modules/docs/arrow-docs/docs/docs/arrow-core/quickstart/setup/README.md

```diff
@@ -39,7 +39,7 @@ allprojects {
 
 Add the dependencies into the project's `build.gradle`:
 
-##### Λrrow Core
+##### Arrow Core
```

Comment:
Do we change to A? There are more cases in this page.

Reply:
Yes, that was proposed by Jason